code/modules/mob/living/living_say.dm

```diff
@@ -277,17 +277,17 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 			var/empty_indexes = get_empty_held_indexes() //How many hands the player has empty
 			if(length(empty_indexes) == 1 || !mute.get_bodypart(BODY_ZONE_L_ARM) || !mute.get_bodypart(BODY_ZONE_R_ARM))
 				message = stars(message)
-			if(length(empty_indexes) == 0)//Both hands full, can't sign
-				to_chat(src, "<span class='warning'>You can't sign with your hands full!</span.?>")
+			if(length(empty_indexes) == 0 || length(empty_indexes) == 1 && !mute.get_bodypart(BODY_ZONE_L_ARM) || length(empty_indexes) == 1 && !mute.get_bodypart(BODY_ZONE_R_ARM))//All existing hands full, can't sign
```

Comment:
Please use () here to make the order of operations more clear.

Reply:
I'd argue against the use of parentheses. In Rust the compiler would warn you against the unnecessary ones, if you added them on a check like this.
`==` has precedence over `||` and `&&`, it's not really ambiguous

Reply:
If parentheses would truly help then I'll add them, but based off the conflicting accounts I'll leave it as is for now.

Reply:
Reading over it again it looks fine, disregard

Reply:
Actually thanks for the suggestion either way, I realized there was a way to shorten the code up and it worked fine in testing.